src/OrderPage/index.js

```diff
@@ -0,0 +1,25 @@
+import './styles.scss';
+import {useEffect, useState} from 'react';
+import Hero from '../Hero';
+import Restaurant from '../Restaurant';
```

Comment:
doesn't look like this component is being used in OrderPage a its greyed out in PHP storm, think this can be removed?